e2e-tests/controller-spark/controller_spark_sql_validation.sh

```diff
@@ -198,12 +198,13 @@ function wait_for_completion() {
   while [[ $(date -u +%s) -le ${end_time} ]]
   do
     local pipeline_status=$(controller "${PIPELINE_CONTROLLER_URL}" status \
-    | sed -n '/{/,/}/p' | jq -r '.pipelineStatus')
+    | sed -n '/^{$/,/^}$/p' | jq -r '.pipelineStatus // ""')
```

Comment:
I think what we are doing here is non-trivial enough to need a comment.

Reply:
Done